security/restricting-fiddler-features.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Restricting Fiddler
+description: "Using the Enterprise policy feature to disable specific Fiddler Everywhere features like remote connections, changing ports, cloud services, and others."
+slug: fe-restrict-policies
+publish: true
+position: 40
+---
+
+# Restricting Fiddler
+
+Users of Fiddler Everywhere often work in a secure corporate environment where specific policies are imposed. That means that administrators need to have options to limit or hardcode particular features of the Fiddler Everywhere application. With the Fiddler Everywhere Enterprise tier, administrators can control a set of functionalities through policy management systems on macOS and Windows.
+
+## Restricting Features on macOS
+
+
+Users on macOS with administrative privileges set policies through the [**defaults**](https://macos-defaults.com/) command and by using the `com.progress-telerik.fiddler` domain alongside the following policies:
```

Comment:
usage of defaults command is not a recommended way - this is our way to test it. 
IT teams who want to set configurations of Fiddler Everywhere can use multiple tools for setting MDM profiles. One such tool is jamf, but there are others as well, like Microsoft Intune.

So, we do not want to explain usage of "defaults" here - we need to state what are the supported key and values and how to use them. 

In addition, com.progress-telerik.fiddler is not a domain, this is our Bundler Identifier. For IT teams, it is important to know the following information for the app, so they can set correct MDM profile:
Team Id: CHSQ3M3P37
App Bundle Id: com.progress-telerik.fiddler
Extension Bundle Id: com.progress-telerik.fiddler.fiddler-extension